__tests__/test.spec.js

```diff
@@ -0,0 +1,70 @@
+const percentCalc = require('../database/helper.js');
+let request = require('supertest');
```

Comment:
This is cool! I will have to look into it. I did not use it in my initial tests.